keps/draft-20180426-security-conformance.md

```diff
@@ -0,0 +1,532 @@
+---
+kep-number: draft-20180426
+title: Security Conformance
+authors:
+  - "@easeway"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-testing
+  - sig-api-machinery
+  - sig-network
+  - sig-node
+  - sig-cluster-lifecycle
+reviewers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+  - "@davidopp"
+approvers:
+  - "@tallclair"
+  - "@ericchiang"
+  - "@liggitt"
+editor: "@easeway"
+creation-date: 2018-04-26
+status: provisional
+---
+
+# Security Conformance
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [Security Conformance Profiles](#security-conformance-profiles)
+    * [Behavioral Criteria](#behavioral-criteria)
+    * [Non-behavioral Criteria](#non-behavioral-criteria)
+* [Graduation Criteria](#graduation-criteria)
+* [Alternatives](#alternatives)
+
+## Summary
+
+Security Conformance defines how a Kubernetes cluster will be considered as secure,
+and also proposes the mechanisms to configure a cluster securely.
+
+## Motivation
+
+Configuring a Kubernetes cluster is complicated,
+and it's very easy to mis-configure a cluster to be insecure.
+The current Kubernetes conformance test suite barely covers security aspects.
+
+### Goals
+
+- A list of behavioral security criteria that can be validated by tests
+- A list of non-behavioral security criteria that can be validated by tests
+
+### Non-Goals
+
+- Mechanisms to satisfy security criteria
+- Non-security related
+- Details about Kubernetes conformance test
```

Comment:
Specifically, this seems to be totally separate from the Kubernetes conformance tests. Maybe it needs another name then?

Reply:
Security Conformance can be validated using tests which will be contributed as part of Kubernetes conformance test.